/**public class Tester
{
  public static void recursivePrint(int num)
  {
    System.out.println("Number:"+num);
    if(num==0) return; else
    recursivePrint(++num);
  }
  public static void main(String[] args)
  {
      Tester.recursivePrint(1);
  }
}

import java.io.*;
class Main
{
  public static void main(String args[])
  #public static void main(String args[])throws FileNotFoundException(used to say the comp that we know that it is going to be an error.
  {
     FileInputStream fx=new FileInputStream("A1.txt");
     
  }
}
class Tester
{
  public static void main(String args[])
  {
      int x=0;
      int y=50/x;
      System.out.println("y =" +y);
  }
}

class Tester
{
  public static void main(String args[])
  {
    try
    {
      int x=0;
      int y=50/x;
      System.out.println("y= " +y);
    }
    catch(ArithmeticException e)
    {
      System.out.println("Division by zero.");
    }
    System.out.println("After catch statement.");
  }
}

class Tester
{
  public static void main(String[] xyz)
  {
    for(int i=0;i<=xyz.length;i++)
    {
      try
      {
       System.out.println(xyz[i]);
      }
      catch(ArrayIndexOutOfBoundsException ar)
      {
        System.out.println("Array out of bounds error: Rectify");
      }
    }
    System.out.println("Thank you"); 
  }
}**/

class Ex1
{
 public static void main(String args[])
 {
    try
    {
       int x=100;
       int y=0;
       int z=x/y;
       System.out.println(args[1]);
    }
    catch(ArrayIndexOutOfRange
}

